adal/src/telemetry/java/com/microsoft/aad/adal/HttpEvent.java

```diff
@@ -120,12 +222,34 @@ public void processEvent(final Map<String, String> dispatchMap) {
             dispatchMap.put(EventStrings.REQUEST_ID_HEADER, "");
         }
 
+        if (dispatchMap.containsKey(EventStrings.SERVER_ERROR_CODE)) {
+            dispatchMap.put(EventStrings.SERVER_ERROR_CODE, "");
+        }
+
+        if (dispatchMap.containsKey(EventStrings.SERVER_SUBERROR_CODE)) {
+            dispatchMap.put(EventStrings.SERVER_SUBERROR_CODE, "");
+        }
+
+        if (dispatchMap.containsKey(EventStrings.TOKEN_AGE)) {
+            dispatchMap.put(EventStrings.TOKEN_AGE, "");
+        }
+
+        if (dispatchMap.containsKey(EventStrings.SPE_INFO)) {
+            dispatchMap.put(EventStrings.SPE_INFO, "");
+        }
```

Comment:
Will this mean that we will send an empty value? Wouldn't it be more reasonable to not send any value in that case to optimize payload size?

Reply:
we can remove it. @iambmelt what do you think?

Reply:
Yeah, let's yank this off. Implementation was written as such to matching behavior of existing fields. Will change the implementation of fields added in this PR, will open an issue for the others 👍